log4j-core/src/test/java/org/apache/logging/log4j/core/config/ConfigurationSourceTest.java

```diff
@@ -30,4 +40,29 @@ public void testJira_LOG4J2_2770_byteArray() throws Exception {
         ConfigurationSource configurationSource = new ConfigurationSource(new ByteArrayInputStream(new byte[] { 'a', 'b' }));
         assertNotNull(configurationSource.resetInputStream());
     }
+
+    /**
+     * Checks if the usage of 'jar:' URLs does not increase the file descriptor
+     * count. Does not work on Windows, since the is no way to count open
+     * descriptors.
+     * 
```

Comment:
Hi @ppkarwasz 
Can't this be tested everywhere by simply trying to copy or delete the jar file?